tools/cp-caps/rcp_caps_test.py

```diff
@@ -80,17 +81,79 @@ def test_diag_commands(self):
         self.__dut.diag_stop()
 
     def test_csl(self):
+        """Test whether the DUT supports CSL transmitter."""
         self.__dataset = self.__get_default_dataset()
         self.__test_csl_transmitter()
 
     def test_data_poll(self):
+        """Test whether the DUT supports data poll parent and child."""
         self.__dataset = self.__get_default_dataset()
         self.__test_data_poll_parent()
         self.__test_data_poll_child()
 
+    def test_throughput(self):
+        """Test Thread network 1 hop throughput."""
+        if not self.__dut.support_iperf3():
+            print('The DUT doesn\'t support the tool "iperf3"')
+            return
+
+        if not self.__ref.support_iperf3():
+            print('The reference device doesn\'t support the tool "iperf3"')
```

Comment:
nit: probably you don't need to quote the `iperf3` so you don't have to escape the single quote.

Like:
```python
print("The reference device doesn't support the tool iperf3")
```